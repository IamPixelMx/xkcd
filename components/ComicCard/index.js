import ComicCard from './ComicCard';

export { ComicCard };
