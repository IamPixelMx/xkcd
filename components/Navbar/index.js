import Navbar from './Navbar';
import NavbarInput from './NavbarInput';
import NavbarItem from './NavbarItem';

export { Navbar, NavbarInput, NavbarItem };
