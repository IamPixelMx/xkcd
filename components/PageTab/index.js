import PageTab from './PageTab';

export { PageTab };
