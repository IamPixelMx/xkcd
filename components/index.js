import { ComicCard } from './ComicCard';
import { Footer } from './Footer';
import { Navbar, NavbarInput, NavbarItem } from './Navbar';
import { PageTab } from './PageTab';
import { Pagination } from './Pagination';

export { ComicCard, Footer, Navbar, NavbarInput, NavbarItem, PageTab, Pagination };
