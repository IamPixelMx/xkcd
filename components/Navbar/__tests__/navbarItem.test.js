import React from 'react';
import renderer from 'react-test-renderer';
import { NavbarItem } from 'components';

describe('NavbarItem', () => {
  const props = {
    route: '/',
    active_route: '/',
    page: 'home',
  };

  const navbarItemElement = renderer.create(<NavbarItem {...props} />).toJSON();
  it('Renders NavbarItem', () => {
    expect(navbarItemElement).toMatchSnapshot();
  });
});
