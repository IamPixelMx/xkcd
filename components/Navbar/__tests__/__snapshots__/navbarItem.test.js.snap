// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarItem Renders NavbarItem 1`] = `
<a
  className="navbar-item is-active"
  href="/"
  onClick={[Function]}
  onMouseEnter={[Function]}
>
  home
</a>
`;
